notifier/senders/msteams/msteams.go

```diff
@@ -44,24 +45,27 @@ type Sender struct {
 	client    *http.Client
 }
 
-// Init initialises settings required for full functionality
-func (sender *Sender) Init(senderSettings map[string]string, logger moira.Logger, location *time.Location, dateTimeFormat string) error {
+// NewSender creates Sender instance.
+func NewSender(senderSettings map[string]string, logger moira.Logger, location *time.Location) (*Sender, error) {
+	sender := &Sender{}
+
 	sender.logger = logger
 	sender.location = location
 	sender.frontURI = senderSettings["front_uri"]
 	maxEvents, err := strconv.Atoi(senderSettings["max_events"])
 	if err != nil {
-		return fmt.Errorf("max_events should be an integer: %w", err)
+		return nil, fmt.Errorf("max_events should be an integer: %w", err)
 	}
 	sender.maxEvents = maxEvents
 	sender.client = &http.Client{
 		Timeout: time.Duration(30) * time.Second, //nolint
```

Comment:
let's move it into const

Reply:
not in func, please :)

Reply:
I just drop `//nolint`, it is not necessary to refactor all Moira code. This const `timeout` is so little thing compare to problems that we have in Moira, so I advise you don't pay attention to it.